compiler+runtime/src/cpp/jank/runtime/core/munge.cpp

```diff
@@ -1,3 +1,5 @@
+#include <regex>
```

Comment:
I left a comment about you including this header in `munge.hpp`, but I also want to now point out that you're *also* including it in `munge.cpp`, (and other files in this PR) which makes the problem even worse. :joy:

Reply:
How does including it into cpp much worse than just including it in hpp?

I understand why including `regex` in hpp would be a problem and also including it in a `cpp` would be worse. But your comment feels like, it's extremely bad on top of just `hpp`?

Could you help me understand ?